bin/pygrb/pycbc_pygrb_page_tables

```diff
@@ -492,50 +450,46 @@ if lofft_outfile or lofft_h5_outfile:
                                               page_size=30)
         kwds = {'title' : "Parameters of loudest offsource triggers",
                 'caption' : "Parameters of the "+
-                            str(min(len(offsource_trigs), opts.num_loudest_off_trigs))+
+                            str(min(len(offsource_trigs),
+                                    opts.num_loudest_off_trigs))+
                             " loudest offsource triggers.  "+
                             "The median reweighted SNR value is "+
                             str(median_bestnr)+
                             ".  The median SNR value is "+
                             str(median_snr),
                 'cmd' :' '.join(sys.argv), }
-        pycbc.results.save_fig_with_metadata(str(html_table), lofft_outfile, **kwds)
+        pycbc.results.save_fig_with_metadata(str(html_table),
+                                             lofft_outfile, **kwds)
 
     # Store BestNR and FAP values: for collective FAP value studies at the
     # end of an observing run collectively
     # TODO: Needs a final place in the results webpage
     # np.savetxt('%s/bestnr_vs_fap_numbers.txt' %(outdir),
     #            full_time_veto_max_bestnr, delimiter='/t')
 
+# Close the trig file
+trig_data.close()
+
 # =======================
 # Load on source triggers
 # =======================
 if onsource_file:
 
     # Get trigs
-    on_trigs = ppu.load_xml_table(onsource_file, "multi_inspiral")
-    logging.info("%d onsource triggers loaded.", len(on_trigs))
-
-    # Separate off chirp mass column
-    on_mchirp = on_trigs.get_column('mchirp')
+    on_trigs = ppu.load_triggers(onsource_file, ifos, None)
+    logging.info("%d onsource triggers loaded.", len(on_trigs['network/event_id']))
 
     # Record loudest trig by BestNR
     loud_on_bestnr = 0
     if on_trigs:
-        on_trigs_bestnrs = ppu.get_bestnrs(on_trigs,
-                                           q=chisq_index,
-                                           n=chisq_nhigh,
-                                           null_thresh=null_thresh,
-                                           snr_threshold=snr_thresh,
-                                           sngl_snr_threshold=sngl_snr_thresh,
-                                           chisq_threshold=new_snr_thresh,
-                                           null_grad_thresh=null_grad_thresh,
-                                           null_grad_val=null_grad_val)
-        loud_on_bestnr_trigs, loud_on_bestnr = \
-            np.asarray([[x, y] for y, x in sorted(zip(on_trigs_bestnrs, on_trigs),
-                                                  key=lambda on_trig: on_trig[0],
-                                                  reverse=True)])[0]
```

Comment:
@jakeb245, you had to adapt this as well in one of your latest PRs.  Can you point us to it, to make sure things are consistent?